userspace/libsinsp/container.h

```diff
@@ -224,13 +232,20 @@ class sinsp_container_manager : public libsinsp::container_engine::container_cac
 	 * This method effectively checks if m_lookups[container_id][ctype]
 	 * exists, without creating unnecessary map entries along the way.
 	 */
-	bool should_lookup(const std::string& container_id, sinsp_container_type ctype) override {
+	bool should_lookup(const std::string& container_id,
```

Comment:
Can you update the comment?
Also, can you expand on why were these changes needed?

Reply:
In the case of more than one cri runtime, we should have a way to address (in this case by index) all the runtimes caches. I'll add a comment.